### PR TITLE
remove unnecessary string from team page + incorrect board; update project info

### LINKED ISSUE #355: Extra "page!" on `/team`
Relatively self-explanatory, see the below screenshot:

Screen Shot 2021-06-30 at 10 51 13 AM

should be a quick fix :)

### PR DESCRIPTION
**Pull request by reginawang99**
closes #355

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -58,6 +62,8 @@
   repo: uclaacm/teach-la-website
   description: "This website! Built with love and HTML, CSS/SASS, and Jekyll (a static site generator)."
   lead:
+    - name: Vivian
+      hash: vha
     - name: Matt
       hash: mattwang
```
Comment (mattxwang) [inline _data/projects.yml:68]: Feel free to drop me haha, I'm just an old man now 😞

Comment (reginawang99) [reply _data/projects.yml:68]: :(

Comment (reginawang99) [inline _data/projects.yml:68]: @vivianha534 / @lumisphere902 any thoughts?

Comment (reginawang99) [reply _data/projects.yml:68]: lemme check w the leo part also :'(

Comment (reginawang99) [reply _data/projects.yml:68]: @vivianha534 / @lumisphere902 any thoughts?

**Comment by matthewcn56** [comment]
hmm do you guys want to add in a previous leads section for our flagship projects like the static site / editor?

**Comment by reginawang99** [comment]
@matthewcn56 that might get a little cluttered there? i think leo and matt are also on past board so idk if thats enough?

File: _data/projects.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@
   link: https://editor.uclaacm.com
   repo: uclaacm/TeachLAFrontend
   lead:
+    - name: Timothy
+      hash: tpoon
     - name: Leo
```
Comment (matthewcn56) [inline _data/projects.yml:11]: Can remove since they're underneath the previous boards section of the teams page.

File: _data/projects.yml [hunk 3]
```diff
@@ -58,6 +62,8 @@
   repo: uclaacm/teach-la-website
   description: "This website! Built with love and HTML, CSS/SASS, and Jekyll (a static site generator)."
   lead:
+    - name: Vivian
+      hash: vha
     - name: Matt
```
Comment (matthewcn56) [inline _data/projects.yml:67]: Can remove since they're underneath the previous boards section of the teams page.

**Review by matthewcn56 — CHANGES_REQUESTED** [review]
Since we have them credited on the `previous boards` section of the team page, I think it's fine to remove Matt @mattxwang and Leo @krashanoff from the leads section of the projects so that people know to contact Timothy for the editor or Vivian for the static site and to remove ambiguity of who the current lead is.

**Comment by matthewcn56** [comment]
> i think leo and matt are also on past board so idk if thats enough?
Ahh I forgot about the past boards page, I think that's fine then!

File: _data/projects.yml (hunk 1 above)
Comment (krashanoff) [reply _data/projects.yml:68]: Feel free to remove me from the current projects listing :+1:

**Comment by reginawang99** [comment]
ok!

File: _data/projects.yml (hunk 1 above)
Comment (reginawang99) [reply _data/projects.yml:68]: ok sounds good!

**Review by matthewcn56 — APPROVED** [review]
Looks good now, ty for making these adjustments!

**Pull request merged**